### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by grilledham**
I think this should be the complete merging of our branches. I'm just going to run some tests to make sure nothing obvious has broken.

### CODE REVIEW

**Comment by grilledham** [comment]
OK I'm happy with this now. You can review and merge now.

**Comment by Valansch** [comment]
51 Files changed. Fun!

File: control.lua [hunk 1]
```diff
@@ -55,7 +56,50 @@ for _, w in pairs(hodor_messages) do
...
+    ['holy'] = true,
+    ['jerk'] = true,
+    ['jesus'] = true,
+    ['petes'] = true,
+    ["pete's"] = true,
+    ['poo'] = true,
+    ['satan'] = true,
+    ['willy'] = true,
+    ['wee'] = true,
+    ['yikes'] = true
```
Comment (Valansch) [inline control.lua]: This list is just ridiculous. Mind if i share it in discord? :D

**Comment by Valansch** [comment]
Fingers crossed

**Pull request merged**